python/ray/serve/_private/proxy.py

```diff
@@ -1196,7 +1196,9 @@ def __init__(
 
         for middleware in http_middlewares:
             self.wrapped_http_proxy = middleware.cls(
-                self.wrapped_http_proxy, **middleware.options
+                self.wrapped_http_proxy,
+                *middleware.args,
+                **middleware.kwargs,
```

Comment:
Thanks Sihan! But this is not backward compatible right? I think we should pin it to >= 0.35.0 or use different args depending on the `starlette` version